### PR TITLE
OADP 1.1 doc changes in 2.7

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/26288

### CODE REVIEW

**Review by birsanv — APPROVED** [review]
lgtm

File: backup_restore/backup_restore_enable.adoc [hunk 1]
```diff
@@ -285,7 +285,7 @@ os get schedules -A | grep acm
 
 Resources are separately backed up in the following groups:
 
-* _Credentials backup_, which contains three backup files for Hive, {product-title-short}, and user-created credentials.
+* _Credentials backup_, which is a backup file storing Hive, {product-title-short}, and user-created credentials and ConfigMaps.
```
Comment (cadawson) [inline backup_restore/backup_restore_enable.adoc]: Does it store Hive or Hive credentials? Depending on the answer, maybe something along the lines of: * _Credentials backup_, which is a backup file that stores Hive credentials, {product-title-short} credentials, and user-created credentials and ConfigMaps.

**Review by cadawson — APPROVED** [review]
One comment. Otherwise, looks good. Thanks!

File: backup_restore/backup_restore_enable.adoc (hunk 1 above)
Comment (birsanv) [reply backup_restore/backup_restore_enable.adoc]: @cadawson it stores Hives credentials

Comment (oafischer) [reply backup_restore/backup_restore_enable.adoc]: Can you confirm if it's credentials @birsanv?

Comment (oafischer) [reply backup_restore/backup_restore_enable.adoc]: Thanks @birsanv for confirming! I'll update and then merge.

**Pull request merged**